packages/frontend/src/pages/AdminLogin.tsx

```
@@ -137,247 +109,86 @@
 
-            {/* Version Text */}
-            <Text fontSize="2xs" color="fg.muted" textAlign="center">
-              Version 2.0.1 | © 2025 DRK
-            </Text>
-          </VStack>
-        </VStack>
-      </Box>
-    </Box>
+                <form.Subscribe
+                  selector={(state) => [state.canSubmit, state.isSubmitting]}
+                  children={([canSubmit, isSubmitting]) => (
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)